lms/templates/problem_notifications.html

```diff
@@ -3,8 +3,8 @@
 <%! from django.utils.translation import ugettext as _ %>
 
 <div class="notification ${notification_type} ${'notification-'}${notification_name}"
-     tabindex="-1">
-    <span class="icon fa fa-${notification_icon}" aria-hidden="true"></span>
+     tabindex="-1" ${'style="display: none"'if notification_name == 'error' else '' } >
```

Comment:
I don't think it is our best practice to include style directly in the HTML. I would prefer you use a class. "is-hidden" is what we commonly use (and general CSS for it may already exist).